docs/book/src/getting-started.md

```diff
@@ -208,7 +208,7 @@ return ctrl.Result{RequeueAfter: nextRun.Sub(r.Now())}, nil
 
 When a Custom Resource is applied to the cluster, there's a designated controller to manage the Memcached Kind. You can check its reconciliation implemented:
 
-From `testdata/project-v4-with-deploy-image/internal/controller/memcached_controller.go`:
+From `$GOPATH/memcached-operator/internal/controller/memcached_controller.go`:
```

Comment:
We have this example in the dir `testdata/project-v4-with-deploy-image/internal/controller/memcached_controller.go` we might create a link.